src/Query/LaravelExpressionProvider.php

```diff
@@ -202,12 +202,15 @@ public function onPropertyAccessExpression($expression)
      * Call-back for function call expression.
      *
      * @param \POData\UriProcessor\QueryProcessor\FunctionDescription $functionDescription Description of the function
-     * @param array<string>                                           $params              Paameters to the function
+     * @param array<string>                                           $params              Parameters to the function
      *
      * @return string
      */
     public function onFunctionCallExpression($functionDescription, $params)
     {
+        if (!isset($functionDescription)) {
+            throw new \InvalidArgumentException('onFunctionCallExpression');
+        }
```

Comment:
If $functionDescription is null, line immediately below (switch $functionDescription->name) loses plot with a null dereference